arch/arc/core/mpu/Kconfig

```
@@ -32,8 +32,8 @@ config ARC_MPU
 	select SRAM_REGION_PERMISSIONS
 	select ARC_CORE_MPU
 	select THREAD_STACK_INFO
-	select GEN_PRIV_STACKS if ARC_MPU_VER != 4
-	select MPU_REQUIRES_POWER_OF_TWO_ALIGNMENT if ARC_MPU_VER !=4
-	select MPU_REQUIRES_NON_OVERLAPPING_REGIONS if ARC_MPU_VER = 4
+	select GEN_PRIV_STACKS if (!(ARC_MPU_VER = 4 || ARC_MPU_VER = 8))
```

We can drop these extra parentheses to make it more readable - there is no need for these parentheses here.

```suggestion
	select GEN_PRIV_STACKS if !(ARC_MPU_VER = 4 || ARC_MPU_VER = 8)
```